src/components/LeftSidebar/NewGroupConversation/SetContacts/SetContacts.vue

```diff
@@ -295,5 +296,9 @@ export default {
 	flex: 0 240px;
 	flex: 1 0 auto;
 	align-content: flex-start;
+
+	&__bubble {
+		overflow: hidden;
```

Comment:
I think, we could put these styles directly to the component. It's local, so there is nothing to upstream =) 

https://github.com/nextcloud/spreed/blob/main/src/components/LeftSidebar/NewGroupConversation/SetContacts/ContactSelectionBubble/ContactSelectionBubble.vue#L98